dashy/src/app/dashboard/widget/divider-widget/divider-widget.component.ts

```diff
@@ -0,0 +1,23 @@
+import {Component} from '@angular/core';
+import {Widget} from '../widget';
+import {WidgetService} from '../../shared/widget.service';
+
+@Component({
+  selector: 'divider-widget',
+  templateUrl: './divider-widget.component.html',
+  styleUrls: ['./divider-widget.component.css']
+})
+export class DividerWidgetComponent extends Widget {
+
+  constructor(private wS: WidgetService) {
+    super(wS);
+  }
+
+  updateData(data: any[]) {
```

Comment:
Remove unimplemented method, as this widget does not update any data and the default implementation does the same.